YAPA/Settings.xaml.cs

```diff
@@ -115,6 +121,41 @@ public Settings(IMainViewModel host, double currentOpacity, Brush currentTextCol
             });
         }
 
+        /// <summary>
+        /// Overrides Windows.Closing-Event
+        /// If windows is closed with Alt+F4 ask the user if he really want to dismiss the date
+        /// The MsgBox is not shown, if the Settings window was Saved() or Dismissed()
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        void Settings_Closing(object sender, CancelEventArgs e)
+        {
+#if DEBUG
+            System.Windows.MessageBox.Show("Closing called");
+#endif
+            // If data is dirty, notify user and ask for a response
```

Comment:
Not sure if pragma Debug is needed, message box can be replaced with Debug.WriteLine(...)